tests/support/client.py

```diff
@@ -49,3 +49,8 @@ def send_slack_message_shortcut(self, data, invalid_token=False):
         assert response.status_code == (401 if invalid_token else 200)
 
         return response.json()
+
+    def login(self, secret, invalid_secret=False):
+        response = requests.post(f"{self.root}/api/login", json={"secret": secret})
+        assert response.status_code == (401 if invalid_secret else 200)
```

Comment:
I wouldn't mix stuff like that together, I think this function should throw an exception if the return code isn't 200